pkg/url/url.go

```diff
@@ -33,33 +36,57 @@ func Delete(client *occlient.Client, urlName string, applicationName string) err
 }
 
 // Create creates a URL
-func Create(client *occlient.Client, urlName string, componentName, applicationName string) (*URL, error) {
+// portNumber is the target port number for the route and is -1 in case no port number is specified
```

Comment:
it would be better if this mentions that `-1` means that the port will be automatically detected from the component